packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -371,115 +528,30 @@ export const Webchat = forwardRef((props, ref) => {
-                    onClick={closeMenu}
-                    url={e.url}
-                    webview={e.webview}
-                    payload={e.payload}
-                    key={i}
-                  >
-                    {Object.values(e.label)}
-                  </Button>
-                )
-              })}
-              <Button onClick={closeMenu}>Cancel</Button>
-            </PersistentMenu>
-          )}
-          {!webchatState.handoff && (
+          {webchatHeader()}
+          {webchatState.error.message ? (
             <div
               style={{
+                flex: '1 1 auto',
                 display: 'flex',
-                borderTop: '1px solid rgba(0, 0, 0, 0.4)'
+                backgroundColor: 'white',
+                alignItems: 'center',
+                justifyContent: 'center',
+                fontFamily: 'Arial, Helvetica, sans-serif'
               }}
             >
-              {props.persistentMenu && (
-                <div
-                  style={{
-                    display: 'flex',
-                    flex: 'none',
-                    width: 50
-                  }}
-                >
-                  <div style={{ width: 50 }}>
-                    <img
-                      style={{
-                        paddingTop: '20px',
-                        paddingBottom: '15px',
-                        marginLeft: '18px',
-                        marginRight: '8px',
-                        cursor: 'pointer'
-                      }}
-                      src={staticAssetsUrl + LogoMenu}
-                      onClick={() => handleMenu()}
-                    />
-                  </div>
-                </div>
-              )}
-              <Textarea
-                name='text'
-                minRows={2}
-                maxRows={4}
-                wrap='soft'
-                maxLength='1000'
-                placeholder={webchatState.theme.textPlaceholder}
-                autoFocus={location.hostname === 'localhost'}
-                inputRef={textArea}
-                onKeyDown={e => onKeyDown(e)}
-                style={{
-                  display: 'flex',
-                  paddingLeft: '10px',
-                  fontSize: 14,
-                  border: 'none',
-                  resize: 'none',
-                  overflow: 'auto',
-                  outline: 'none',
-                  marginTop: '13px'
-                }}
-              />
+              Error: {webchatState.error.message}
             </div>
-          )}
-          {webchatState.webview && (
-            <RequestContext.Provider value={webviewRequestContext}>
-              <WebviewContainer
-                style={{
-                  ...props.theme.webviewStyle
-                }}
-                webview={webchatState.webview}
-              />
-            </RequestContext.Provider>
+          ) : (
+            <>
+              {webchatMessageList()}
+              {webchatState.replies &&
```

Comment:
The replies, is going to be rendered fixed at the end of the message list. It should be as a new message that goes away when scrolling

Reply:
I'll leave it like this for now. I'd like to add a Messenger-like behaviour: replies are fixed at the end but they disappear if you scroll up more than X%